src/components/Tooltip/Tooltip.scss

```diff
@@ -28,17 +28,19 @@ $content-max-width: rem(200px);
 
 .light {
   .Wrapper {
-    background: color('white');
+    background: var(--p-surface-light-foreground, color('white'));
+    // border: border-width() solid var(--p-border-subdued-on-surface, transparent);
```

Comment:
This is similar to the `border` function:

https://github.com/Shopify/polaris-react/blob/01be565bcc4d8c7babdfc8e9fbfac9842fd9536c/src/styles/foundation/_borders.scss#L2

I didn't use the function since it we don't want a border if we are not using the color system. The only difference is that the function falls back on `white`, whereas this uses `transparent`.

Reply:
(Using this approach instead of `box-shadow: shadow(deep)` which visually adds a border + a shadow)